nhost/service/manager.go

```diff
@@ -70,6 +71,14 @@ type dockerComposeManager struct {
 	env           []string
 }
 
+func (m *dockerComposeManager) HasuraConsoleCmd(ctx context.Context) (*exec.Cmd, error) {
+	if m.hc == nil {
```

Comment:
is there a good reason why this may happen? It might be better to ensure it is always initialized instead (i.e. by ensuring `NewDockerComposeManager` function takes care of it. Unless there is a good reason to initialize it "manually" later on.